ansible/group_vars/all

```diff
@@ -106,7 +106,7 @@ nginx_vhosts:
         fastcgi_param PATH_INFO $path_info;
         fastcgi_param PATH_TRANSLATED $document_root$path_info;
         include snippets/fastcgi-php.conf;
-        fastcgi_pass unix:/var/run/php5-fpm.sock;
+        fastcgi_pass {{ fastcgi_pass_sock }};
```

Comment:
Can we not have the same path for the socket across versions e.g. `unix:/var/run/php-fpm.sock`? A connectbox can’t have php5 and php7 running simultaneously (because 5 isn’t supported on all platforms), so having the same path for the socket doesn’t seem to have any downside.